source/agora/consensus/data/UTXOSet.d

```diff
@@ -27,6 +27,51 @@ import std.file;
 
 mixin AddLogger!();
 
+/// The structure of spendable transaction output
+public struct UTXOSetValue
+{
+    /// Height of the block to be activated
+    ulong height;
```

Comment:
This could use a more descriptive name.
I can suggest "heightLock", "frozenUntil", "lockedUntil", "minHeight"... But I'm open to suggestion @TrustHenry @AndrejMitrovic @MukeunKim

Reply:
I suggest snake_case.

Reply:
I suggest "height_lock".

Reply:
I suggest "locked_until"~

Reply:
I would go with @MukeunKim's suggestion, or maybe `unlock_height`

Reply:
I like `unlock_height`

Reply:
Me too `unlonk_height`  because  `height + 1;`